stdlib/strings/strings.go

```diff
@@ -411,7 +411,7 @@ func init() {
 	runtime.RegisterPackageValue("strings", "lastIndex",
 		generateDualArgStringFunctionReturnInt("lastIndex", []string{stringArgV, substr}, strings.LastIndex))
 	runtime.RegisterPackageValue("strings", "lastIndexAny",
-		generateDualArgStringFunctionReturnInt("lastIndexAny", []string{stringArgV, substr}, strings.LastIndexAny))
+		generateDualArgStringFunctionReturnInt("lastIndexAny", []string{stringArgV, chars}, strings.LastIndexAny))
```

Comment:
If I'm reading this right, this changes the parameter name so it'd constitute a breaking change :cold_sweat: 
Are we missing a test that should have been failing up until now?

Reply:
You could probably call this a "fixing change" 😄 . The function signature specified a `chars` parameter as does the go `strings.lastIndexAny` function. The misnamed Go param caused the function to fail. I don't think we have a specific test for this function, but as far as I can tell, it has never actually worked. This fixes it.

Reply:
:muscle: okay great. Upsetting to find out things aren't wired up correctly after however long, but this isn't surprising. I think `strings` in particular is a module in need of polish.